### ISSUE TITLE
Save models

### ISSUE THREAD

**Issue by pfistfl** [mlr-org/mlr3keras#13]
Test and check that models can be safely serialized.
Export / Offer a "save" function.

**Comment by pfistfl** [comment]
See https://github.com/mlr-org/mlr3pipelines/issues/297

**Comment by pfistfl** [comment]
Implemented and tested for normal learners, waiting on the above issue for pipelines.

Additionally: Test whether learners can be serialized / unserialized.

**Comment by pfistfl** [comment]
Will work for pipelines upon update, serialization is out-of-scope for now.
